### PR TITLE
Flaky test: CentralDogmaBeanTest.test()

### PR DESCRIPTION
**Pull request by jmostella**
* CentralDogmaBeanTest newline trigger CI, in order to observe failure.

### CODE REVIEW

**Comment by trustin** [comment]
Why is it so?

**Comment by jmostella** [comment]
don't know yet :) trying to get reproducible PR.

File: it/src/test/java/com/linecorp/centraldogma/it/updater/CentralDogmaBeanTest.java [hunk 1]
```diff
@@ -67,18 +70,23 @@ public void setup() {
...
     }
 
+    @Test
+    public void stayDefault() {
+        final TestPropertyDefault property = factory.get(new TestPropertyDefault(), TestPropertyDefault.class);
+        await().pollDelay(500, TimeUnit.MILLISECONDS)
+               .until(() -> property.getFoo() == 10 &&
+                            "20".equals(property.getBar()) &&
+                            property.getQux().equals(ImmutableList.of("x", "y", "z")) &&
+                            Objects.isNull(property.getRevision()));
```
Comment (trustin) [inline it/src/test/java/com/linecorp/centraldogma/it/updater/CentralDogmaBeanTest.java]: `property.getRevision() == null`

File: it/src/test/java/com/linecorp/centraldogma/it/updater/CentralDogmaBeanTest.java [hunk 2]
```diff
@@ -67,18 +70,23 @@ public void setup() {
         factory = new CentralDogmaBeanFactory(dogma.client(), objectMapper);
     }
 
+    @Test
+    public void stayDefault() {
+        final TestPropertyDefault property = factory.get(new TestPropertyDefault(), TestPropertyDefault.class);
+        await().pollDelay(500, TimeUnit.MILLISECONDS)
```
Comment (trustin) [inline it/src/test/java/com/linecorp/centraldogma/it/updater/CentralDogmaBeanTest.java]: OK to omit `pollDelay()`

**Comment by jmostella** [comment]
Thanks! Updated

**Comment by trustin** [comment]
Thanks, @jmostella !

**Pull request merged**